spec/unit/file_serving/configuration/parser_spec.rb

```diff
@@ -98,32 +98,28 @@ def write_config_file(content)
     end
 
     [:allow,:deny].each { |acl_type|
-      it "should support inline comments in #{acl_type}" do
+      it "should ignore inline comments in #{acl_type}" do
         write_config_file "[one]\n#{acl_type} something \# will it work?\n"
 
-      expect(@mount).to receive(:info)
-      expect(@mount).to receive(acl_type).with("something")
-      @parser.parse
+        @parser.parse
       end
 
-      it "should tell the mount to #{acl_type} from ACLs with varying spacing around commas" do
+      it "should ignore #{acl_type} from ACLs with varying spacing around commas" do
         write_config_file "[one]\n#{acl_type} someone,sometwo, somethree , somefour ,somefive\n"
 
-        expect(@mount).to receive(:info).exactly(5).times
-        expect(@mount).to receive(acl_type).exactly(5).times.with(eq('someone').or eq('sometwo').or eq('somethree').or eq('somefour').or eq('somefive'))
         @parser.parse
       end
```

Comment:
Are these testing anything anymore at all? Or can they be removed completely?

Reply:
I left them in to make sure we still parse `fileserver.conf` with allow/deny rules in them, as could occur during an upgrade. Currently we log an error and continue. But I should probably update the test to expect the error message.

I also considering having the parser raise if it encountered an allow or deny rule. But I feel like that would bit us during upgrades.

Reply:
ahh, yeah, I think it would be clearer to future us if you added an expectation.

Reply:
Ah so I kept the tests above to show that we could still parse the same inputs as before and that the `acl_type` (either `allow` or `deny` is ignored). The tests after those that [verify the error is logged as expected](https://github.com/puppetlabs/puppet/pull/8320/files#diff-483269bc1e381cfdc995bc8e7b421b89R119).

Reply:
I still think it looks weird to have tests that don't have any expectations.. 
Maybe just `expect { @parser.parse }.not_to raise_error`?

Reply:
Expecting the test not to raise is a bit redundant (since rspec will fail the test if it does). But to your point, it'd be clearer to expect that `@parser.parsrer` returns the expected mounts.

Reply:
Updated

Reply:
I didn't see any update.. Is my stuff out of date or did you forget to push?

Reply:
Oh jeez... looks like I didn't push... The diff is just the test changes, so I'll submit a different PR for that:

```diff
$ git diff 2d60297d00..c64418b370
diff --git a/spec/unit/file_serving/configuration/parser_spec.rb b/spec/unit/file_serving/configuration/parser_spec.rb
index 72116de7a5..32dbfd3268 100644
--- a/spec/unit/file_serving/configuration/parser_spec.rb
+++ b/spec/unit/file_serving/configuration/parser_spec.rb
@@ -101,13 +101,14 @@ describe Puppet::FileServing::Configuration::Parser do
       it "should ignore inline comments in #{acl_type}" do
         write_config_file "[one]\n#{acl_type} something \# will it work?\n"
 
-        @parser.parse
+        expect(@parser.parse).to eq('one' => @mount)
+
       end
 
       it "should ignore #{acl_type} from ACLs with varying spacing around commas" do
         write_config_file "[one]\n#{acl_type} someone,sometwo, somethree , somefour ,somefive\n"
 
-        @parser.parse
+        expect(@parser.parse).to eq('one' => @mount)
       end
 
       it "should log an error and print the location of the #{acl_type} rule" do
```